### PR TITLE
V1.1 release - Updating version numbers

### PR DESCRIPTION
**Pull request by RFMVasconcelos**
As requested by @jlewi in #1984. 

Followed steps 2 to 4 in [instructions](https://github.com/kubeflow/kubeflow/blob/master/docs_dev/releasing.md#updating-version-numbers-etc-for-the-upcoming-release-major-minor-or-patch).

### CODE REVIEW

File: layouts/shortcodes/config-file-gcp-basic-auth.html [hunk 1]
```diff
@@ -1 +1 @@
-kfctl_gcp_basic_auth.v1.0.2.yaml
\ No newline at end of file
+kfctl_gcp_basic_auth.v1.1.0.yaml
```
[the remark this answers is not in the document]
Comment (jlewi) [reply layouts/shortcodes/config-file-gcp-basic-auth.html:1]: @Jeffwan Should platform owners be responsible for updating the short codes as part of updating their docs?

File: config.toml [hunk 2]
```diff
@@ -171,6 +171,11 @@ githubbranch = "v1.0-branch"
   githubbranch = "v0.7-branch"
   url = "https://v0-7.kubeflow.org"
 
+[[params.versions]]
+  version = "v1.0"
+  githubbranch = "v1.0-branch"
+  url = "https://v1-0.kubeflow.org"
```
Comment (jlewi) [inline config.toml:177]: This URL doesn't appear to be live at. What does the release guide say about when this gets created?

Comment (RFMVasconcelos) [reply config.toml:177]: The release guide does not mention adding this, but the config.toml file says that here is where we add new versions to website drop-down. 

Given that we do not have the URL live yet, we need to to [this](https://github.com/kubeflow/kubeflow/blob/master/docs_dev/releasing.md#creating-and-publishing-a-website-branch-for-vxy) step first and then come and add this.

I'll delete this last commit, so we can merge and we'll do this step when we're ready.

File: layouts/shortcodes/config-file-gcp-basic-auth.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jeffwan) [reply layouts/shortcodes/config-file-gcp-basic-auth.html:1]: @jlewi  I think so, currently, different platform's progress and solution are different, I think it's better for platform owner to update manifest. release owner can take care of v1.1 general contents in this case. Update manifest tag is kind of risky for release owner.

[the remark this answers is not in the document]
Comment (Jeffwan) [reply layouts/shortcodes/config-file-gcp-basic-auth.html:1]: It also helps reduce the heavy burdens on release owners.

[the remark this answers is not in the document]
Comment (jlewi) [reply layouts/shortcodes/config-file-gcp-basic-auth.html:1]: Makes sense to me. @RFMVasconcelos; I think what @Jeffwan is suggesting is that in this PR we revert the changes to the platform specific shortcodes. The platform OWNERs can then update those as part of PRs that would update their docs pages in order to remove the "these docs are outdated banner."

[the remark this answers is not in the document]
Comment (jlewi) [reply layouts/shortcodes/config-file-gcp-basic-auth.html:1]: So I think just revert the other files and then this is good to go.

Might be worth updating the release guide to capture this for future releases.

[the remark this answers is not in the document]
Comment (RFMVasconcelos) [reply layouts/shortcodes/config-file-gcp-basic-auth.html:1]: Sounds good to me! 

To clarify, which files are good to be updated to v1.1?
> Config-file-xxx:
- gcp
- aws
> Config-uri-xxx:
- gcp
- aws

Then we leave the others unchanged?

[the remark this answers is not in the document]
Comment (jlewi) [reply layouts/shortcodes/config-file-gcp-basic-auth.html:1]: I think we want to revert everything except config.toml

**Comment by jlewi** [comment]
@8bitmp3 I believe @RFMVasconcelos is out this week. Any chance you could take over this PR so we can get it merged? We basically just need the changes in config.toml but nothing else.

**Comment by RFMVasconcelos** [comment]
@jlewi done :)

**Comment by jlewi** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**